content/en-us/tutorials/scripting/basic-scripting/scoring-points.md

```diff
@@ -121,16 +121,16 @@ Test your experience and you should see the leaderboard appear in the top right
 
 ## Counting Time
 
-Each user should earn a point for each second they are alive. A `while` loop and the `wait` function can be used to update the value of points every second.
+Each user should earn a point for each second they are alive. A `while` loop and the `task.wait` function can be used to update the value of points every second.
```

Comment:
Adding `Libary.` makes the code automatically link to the [reference page](https://create.roblox.com/docs/reference/engine/libraries/task#wait) (see [STYLE.md](https://github.com/Roblox/creator-docs/blob/main/STYLE.md#links)):

```suggestion
Each user should earn a point for each second they are alive. A `while` loop and the `Library.task.wait()` function can be used to update the value of points every second.
```